recipes/bioconductor-dexseq/meta.yaml

```diff
@@ -49,6 +49,7 @@ requirements:
     - r-rcolorbrewer
     - r-statmod
     - r-stringr
+    - python 2*
```

Comment:
can you just use `python` here and skip the build for [py3k or osx]. This will solve your issue.